build/build.xml

```diff
@@ -101,6 +108,17 @@ External Dependencies:
 		<property name="src.isInit" value="true" />
 	</target>
 
+	<!-- create script to run built commandbox using sources (for core cfml development) -->

```

Comment:
This is just because I'm lazy, and don't want to have to remember how to do it.

All it does is create a script file in the dist dir that you can use after running a build to then test it-- the main thing being that it uses the source code in /src/cfml and a temporary commandbox_home.

I'd heard some folk were running a build, installing, then editing the source code in ~{home}/.CommandBox to do development, where as if you just run the bootstrap you can edit stuff right where it is-- which is what I do, but sometimes I have to search my brain for a second to come up with the proper command.